### ISSUE TITLE
Broken gamepad axis handling

### ISSUE THREAD

**Issue by ndarilek** [PradeepKumarRajamanickam/kurinji#40]
Windows 10, Xbox Wireless controller connected via USB. I've verified controller functionality in Godot, and button presses in examples seem to work.

> cargo run --example gamepad_with_code

seems to work fine for button-presses, but axis events don't work at all.

I did a bit of debugging, and tweaked your code like so:

```
    pub(crate) fn get_gamepad_from_player_handle(self, player: usize) -> Option<Gamepad> {
        println!("{:?}", self.player_to_joystick_map);
        return match self.player_to_joystick_map.get(&player) {
```

This map always seems to be empty, meaning this match would fail:

```
    pub(crate) fn gamepad_axis_system(
        mut input_map: ResMut<Kurinji>,
        pad_axis: Res<bevy::input::Axis<bevy::input::gamepad::GamepadAxis>>,
    ) {
        for (k, v) in input_map.clone().joystick_axis_binding.iter() {
            let player = k.0;
            let axis = k.1.clone();
            let is_positive = Kurinji::is_gamepad_axis_positive(axis.clone());
            println!("Here");

            if let Some(bevy_gamepad) = input_map.clone().get_gamepad_from_player_handle(player) {
```

So I don't think this is a platform issue. Seems this mapping isn't correctly set up in either the example or in my own code, which doesn't run multiple players through one gamepad. Not sure what the correct fix is, since I don't completely get the intent of routing multiple players to a single gamepad. Why not use a gamepad per player?

Thanks.

**Comment by PradeepKumarRajamanickam** [comment]
Hey there,

Yep the issue is reproducible. When this code was written ( a long long while ago) bevy gamepad api was in early development and axis event apis did not exist. With 0.3.0 things have change, we can now use events to listen for axis changes. We are not actually routing multiple player to a single pad, it is gamepad per player. It is just not obvious because of the limitations of the old api. It is time to update this code.

**Comment by PradeepKumarRajamanickam** [comment]
@ndarilek I have pushed a fix to master. Can you please verify and let me know if it solved the issue?

**Comment by ndarilek** [comment]
Haven't checked the values for sanity, but it does indeed get events now 
when I move the left/right sticks. Thanks!

**Comment by PradeepKumarRajamanickam** [comment]
Fix available on Kurinji v1.0.4.
Closing ticket.
